src/app/event/_components/EventTop.module.scss

```
@@ -0,0 +1,17 @@
+.container {
+  width: 100%;
+  height: 90rem;
+  background: url('../../../../public/images/backconfetti.png'), linear-gradient(to left, #d3e1ff, #fcfcf2);
```

요거 위에서 import 먼저 하시는건 어떠신가욤??
```suggestion
 import backConfetti from '../../../../public/images/backconfetti.png';
 
  background: url(backConfetti), linear-gradient(to left, #d3e1ff, #fcfcf2);
```

---

css도 이미지를 import 해올수 있나용???